apex/contrib/csrc/groupbn/batch_norm_add_relu.h

```diff
@@ -408,7 +541,11 @@ class NhwcBatchNormAddRelu {
   // Calculate the expected bwd kernel occupancy, as dictated by shared memory usage.
   static int smem_driven_bwd_occupancy(int device_id, const int max_cta_per_sm) {
     using namespace at::cuda::utils;
+#ifdef __HIP_PLATFORM_HCC__
+    int bwd_reduction_bytes = THREADS_PER_PIXEL*(THREADS_PER_CTA/64)*ELEMENTS_PER_LDG*sizeof(float);
+#else
     int bwd_reduction_bytes = THREADS_PER_PIXEL*(THREADS_PER_CTA/32)*ELEMENTS_PER_LDG*sizeof(float);
+#endif
```

Comment:
Ditto about WARP_SIZE

Reply:
@micmelesse @jithunnair-amd @jeffdaily I updated the code to use C10_WARP_SIZE here too